.github/workflows/build-and-push.yml

```diff
@@ -27,7 +27,16 @@ jobs:
         uses: actions/checkout@v4
         with:
           fetch-depth: 0
-          token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Show branch and commit info
```

Comment:
This is to show `branch` information for every checkout operation, in each of the workflows. WIthout this, it was tedious to figure out which branch is the checkout done from.